### ISSUE TITLE
Question: how to contribute suggestion to fiware openapi specs?

### ISSUE THREAD

**Issue by ioggstream** [Fiware/specifications#10]
Questions: 

  - @jason-fox @fgalan on which repo can I contribute / PR to fiware openapi specs?
  - are those specs just the autosync/conversion from another source?

**Comment by jmcanterafonseca** [comment]
@ioggstream Yes, the specifications come from Apiary Specifications and we just use the ApiMatic tool to translate. It could be a good idea to have native OpenAPI 3.0 specs, and a contribution on that respect will be very welcome

The repo on which to make a PR would be this one
